apps/client/src/app/services/data.service.ts

```diff
@@ -686,6 +687,10 @@ export class DataService {
     return this.http.get<Tag[]>('/api/v1/tags');
   }
 
+  public fetchWatchlist() {
+    return this.http.get<AssetProfileIdentifier[]>('/api/v1/watchlist');
```

Comment:
I have noticed that the watchlist endpoint directly returns `AssetProfileIdentifier[]`. Could you change it to an object:

```
{ watchlist: AssetProfileIdentifier[] }
```

This would make it easier to potentially extend the response by e.g. an error field without breaking the interface.